easybuild/tools/module_generator.py

```diff
@@ -213,6 +214,79 @@ def swap_module(self, mod_name_out, mod_name_in, guarded=True):
         """
         raise NotImplementedError
 
+    def _generate_help_text(self):
+        """
+        Generate syntax-independent help text used for `module help`.
+        """
+        # General package description (mandatory)
+        description = self.app.cfg['description']
+        lines = [
+            '',
+            'Description',
+            '===========',
+            "%s" % description.strip(),
+        ]
```

Comment:
this is a common pattern below, so maybe define a helper function inside of this method to ensure consistency of the format?

```python
def _generate_help_text(self):
    ...

    def generate_section(sec_name, sec_txt):
        """
        Generate section with given name and contents
        """"
        if sec_txt:
            return [
                '',
                sec_name,
                '=' * len(sec_name),
                sec_txt.strip(),
             ]
         else:
             return []

        # General package description (mandatory)
        lines = generate_section('Description', self.app.cfg['description'])

        # Package usage instructions (optional)
        lines.extend('Usage', self.app.cfg['usage'])

        # Additional information: homepage + (if available) doc paths/urls, support info, contacts, ...
        lines.extend('More information', " - Homepage: %s" % self.app.cfg['homepage'])
        ...

        extensions = ', '.join(sorted(['%s-%s' % (ext[0], ext[1]) for ext in exts_list], key=str.lower))
        lines.extend('Extensions', '\n'.join(wrap(extensions, 78)))
```